.python-version

```diff
@@ -0,0 +1 @@
+3.12
```

Comment:
Are we already using pyenv?

Reply:
I am, it's a convenient helper. I was going to open a new PR with more creature comforts for development, but this doesn't really hurt to have already in there IMO. I'll delete it if you're against it.

Reply:
I wanna use it. Put whenever is meant to go into the project for it. But maybe do a dedicated PR with some instructions for noobs.